### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by katiegoines**
#### Description of changes:
Adding a `ready to merge` checkbox for contributors to check when the PR has received the necessary approvals, checks have passed, and merge conflicts have been resolved. Also correcting a spelling error

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [x] amplify-librarires

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br />
      _ref: MDX: `[link](https://link.com)`
            HTML: `<a href="https://link.com">link</a>`_

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -31,5 +31,8 @@ Which platform(s) are affected by this PR (if applicable)?
 - [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br />
       _ref: MDX: `[link](https://link.com)`
             HTML: `<a href="https://link.com">link</a>`_
+            
+### When this PR is ready to merge, please check the box below
+- [ ] ready to merge
```
Comment (timngyn) [inline .github/PULL_REQUEST_TEMPLATE.md:36]: Should this be capitalize like the other "checks"?

**Review by timngyn — APPROVED** [review]
Looks good to me! Just had a small nitpick, not blocking

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (katiegoines) [inline .github/PULL_REQUEST_TEMPLATE.md:36]:
```suggestion
- [ ] Ready to merge
```

Comment (katiegoines) [reply .github/PULL_REQUEST_TEMPLATE.md]: Good callout - fixed. Thanks!

**Review by timngyn — APPROVED** [review]

**Review by rachelnabors — APPROVED** [review]

**Pull request merged**